index.d.ts

```diff
@@ -0,0 +1,760 @@
+// Definitions by James George (https://github.com/jamesgeorge007)
+
+export function cli(e: any): any;
+
+export function main(e: any, t: any): any;
+
+interface Config {
```

Comment:
I think some properties fo the config is missing right? Maybe it's nice to add these too:
- Managers
- Utilities
- Servers
- Virtualization
- SDKs
- IDEs
- Languages
- Databases

Reply:
Btw, we can also set all properties to "possible undefined" in the interface itself ofc, to [fix this one](https://github.com/tabrindle/envinfo/pull/105#discussion_r355190370). e.g.

```
interface Config {
    System?: string[];
    ...
}
```